### PR TITLE
Add support for JavaScript templates in the GUI.

### PR DESCRIPTION
**Pull request by genintho**
screen shot 2014-08-03 at 12 14 59 pm

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -222,7 +228,9 @@
         Source and bugtracker are <a href="http://github.com/kangax/html-minifier">hosted on Github</a>.
       </p>
     </div>
-    <script src="dist/htmlminifier.min.js"></script>
+    <script src="src/htmllint.js"></script>
```
Comment (genintho) [inline index.html]: Need to be reverted before final review

File: src/htmlminifier.js [hunk 2]
```diff
@@ -406,6 +406,17 @@
     return text;
   }
 
+  function minifyJSTemplates(text, options, currentAttrs) {
+      var templatesType = ['text/x-handlebars-template'];
```
Comment (genintho) [inline src/htmlminifier.js]: I am planning to support multiple "type"

**Comment by kangax** [comment]
Take a look at `processScripts` option — https://github.com/kangax/html-minifier/blob/gh-pages/README.md

**Comment by genintho** [comment]
@kangax thanks.

Pull request updated.

**Comment by kangax** [comment]
Awesome, thanks!

**Pull request merged**